addons/website_sale/controllers/main.py

```diff
@@ -233,6 +233,11 @@ def _get_search_options(
             'display_currency': pricelist.currency_id,
         }
 
+    def _get_additional_shop_values(self, values):
+        """ Hook to update values used for rendering website_sale.products template """
+        return values
```

Comment:
We already update values with the return of this function, so either we return an empty dict here or we modify values in place. I originally gave values as argument as we might require some information from values to deduct some other stuff.
```suggestion
        return {}
```

Reply:
@Williambraecky fixed. Hope now it is ok!